client/tests/stubs.js

```diff
@@ -46,6 +46,10 @@ global.wagtailConfig = {
 
 global.wagtailVersion = '1.6a1';
 
+global.wagtail = {
+  ui: {},
```

Comment:
We're exposing `ui` in the stubs, but it looks like we're not using this anywhere?

Reply:
No – like for `wagtailVersion`, this is just trying to replicate the global variables set elsewhere. We could remove this if you think it is confusing since it is unused?

Reply:
Maybe if there is no reason to replicate the `ui` bit then just it leave it be an empty dictionary?

Reply:
👍 check